design-collaboration-guidelines.md

```diff
@@ -0,0 +1,82 @@
+# Design Collaboration Guidelines
+
+As an app developer to provide the best version of the design, you need to define a standard vocabulary between the design team and you.
+
+Designers are more and more integrating design systems as part of their work, and we can benefit from that to layout common ground. Here below we would define the minimum standard requirements to keep consistency and the usuals forgotten piece of app design. Independently of the size of your team, there is already a set of best practices applied by your designer that you can benefit, and other you will need anyway at some point during the development of the app.
+
+* [A COMPREHENSIVE GUIDE TO DESIGN SYSTEMS](https://www.invisionapp.com/blog/guide-to-design-systems/)
+
+## Style guide - Foundation
+These are the basic style that will serve as the foundation of all other components. And avoid mistakes by not using the proper element. Independently if you follow material design, fluent design or any other guidelines, these basics style element will facilitate design adjustment and rebranding.
+### Color Palette
+A list of all the color used in the app with a name associated.
+
+![color palette](./assets/color-palette.png)
+
+* [SwiftGen Colors](https://github.com/SwiftGen/SwiftGen#colors)
+* [Asset catalog colors on Xcode 9](https://blog.zeplin.io/asset-catalog-colors-on-xcode-9-c4fdccc0381a)
+* [Android Resources Color](https://developer.android.com/guide/topics/resources/more-resources.html#Color)
+* [Material Guidelines Color System](https://material.io/guidelines/style/color.html#color-color-system)
+
+### Typography - Fonts
+A list of Fonts and Size and when possible complete style including color and format.
```

Comment:
I'd rewrite this as ``A list of Fonts and Size. When possible, a complete style guide including color and format should be also provided.``